modules/apps/portal-search-elasticsearch7/portal-search-elasticsearch7-impl/src/main/java/com/liferay/portal/search/elasticsearch7/internal/facet/DefaultFacetProcessor.java

```diff
@@ -33,7 +33,10 @@
  * @author Michael C. Han
  * @author Milen Dyankov
  */
-@Component(property = "class.name=DEFAULT", service = FacetProcessor.class)
+@Component(
+	property = "class.name=DEFAULT",
+	service = {DefaultFacetProcessor.class, FacetProcessor.class}
```

Comment:
@W2jx I remember that Tina mentioned we want to avoid declaring the class itself as much as possible in `service` attribute. Please try to find another way to reference it.